### ISSUE TITLE
How to Train for Custom dataset

### ISSUE THREAD

**Issue by FatimaZulfiqar** [hyk1996/Single-Human-Parsing-LIP#13]
Hi. Will you please tell me how can I prepare and train for custom dataset like Market-1501?

**Comment by Yukun-Huang** [comment]
Reorganize your own data following the file structure of the LIP dataset.

**Comment by FatimaZulfiqar** [comment]
Yes but I have downloaded the LIP dataset but I am unable to reorganize Market-1501 dataset. The file structure for both datasets are pretty different. Can you provide me with the script or is there any way to do this? Actually I am trying to create part masks on Market-1501 dataset for Person ReID task

**Comment by Yukun-Huang** [comment]
Human pasing and ReID are two different tasks. You cannot train on Market-1501 because there are no human pasing labels.
Directly using the model weights trained on the LIP dataset is a feasible solution.

**Comment by FatimaZulfiqar** [comment]
ohhh okay thankyou. But I have one more confusion that can weights of model that was originally trained on LIP dataset be tested and evaluated on market-1501 dataset directly? or do I also have to reorganize Market-1501 for testing also?

**Comment by Yukun-Huang** [comment]
Yes, the model trained on LIP can generalized to Market. No need to rereganize Market-1501, directly using inference.py to extract the mask is ok.

**Comment by FatimaZulfiqar** [comment]
ok thank you so much

**Comment by FatimaZulfiqar** [comment]
Hi I am getting following errors. I was hoping maybe you can help me to fix it

![Screenshot 2021-04-01 005352](https://user-images.githubusercontent.com/81231619/113202978-e0d00680-9284-11eb-8056-c0b9bfbff952.png)
![Screenshot 2021-04-01 005441](https://user-images.githubusercontent.com/81231619/113202982-e1689d00-9284-11eb-8b87-c4938d296751.png)

**Comment by SuperMamba** [comment]
Hello, the data set link is invalid.
